CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
     * Add `shopperSessionId` parameter to `ShopperInsightsClient`
 * BraintreePayPal
     * Add `shopperSessionId` to `PayPalCheckoutRequest` and `PayPalVaultRequest`
+    * Replace `sendPayPalPresentedEvent()` and `sendVenmoPresentedEvent()` with `sendPresentedEvent()`
```

Comment:
Not sure if in Android it's similar to iOS - `sendPresentedEvent(for:presentmentDetails:)` instead of `sendPresentedEvent()`